src/runtime/runtime.ts

```diff
@@ -168,10 +168,23 @@ export class Runtime implements SubscriptionsInterface {
     this.buttonApi_.init();  // Injects swg-button stylesheet.
   }
 
+  /**
+   * Returns a promise for the configured runtime. If requested, also begins
+   * configuring the runtime.
+   *
+   * NOTE: Begininning runtime configuration prematurely can interfere with
+   * publisher integrations (b/283281215). This method should only be called
+   * with `startConfiguringRuntime=true` when immediate configuration is
+   * required, after integratros have had an opportunity to begin configuration
+   * using the the `init` method.
+   *
+   * @param startConfiguringRuntime Whether to begin configuring the runtime.
+   * @returns A promise resolving to the configured runtime.
```

Comment:
This line seems a little redundant since the method description starts with the same info

Reply:
I think the TypeScript style guide suggests using `@return` instead of `@returns`, but only when it adds information. In this case, is it adding information though?
https://google.github.io/styleguide/tsguide.html#redundant-comments

Reply:
I see what you mean about there being other details in the description though

Reply:
You're right, updated to `@return`. The style guide says it _may_ be omitted where it is obvious, I don't think it hurts clarity to be explicit here.